src/from.rs

```diff
@@ -1,4 +1,4 @@
-//! Convert arrow-rs objects to an `Robj`
+//! Create arrow-rs objects from a `Robj`
```

Comment:
Can you change `a` to `an`? This is required since "R" starts with a vowel sound.

Reply:
```suggestion
//! Create arrow-rs objects from an `Robj`
```